build.sbt

```diff
@@ -478,14 +491,14 @@ lazy val `scio-sql`: Project = project
   .settings(
     description := "Scio - SQL extension",
     libraryDependencies ++= Seq(
```

Comment:
If these are Java dependencies, it might be nice to have a comment like previously in the build file.